sparse/_compressed/compressed.py

```diff
@@ -237,6 +168,33 @@ def nbytes(self):
             nbytes += self.indptr.nbytes
         return nbytes
 
+    @property
+    def axis_order(self):
+        axis_order = list(self.compressed_axes)
+        axis_order.extend(
+            np.setdiff1d(np.arange(len(self.shape)), self.compressed_axes)
+        )
+        return axis_order
+
+    @property
+    def axisptr(self):
+        # array location where the uncompressed dimensions start
+        return len(self.compressed_axes)
+
+    @property
+    def compressed_shape(self):
+        row_size = np.prod(self.reordered_shape[: self.axisptr])
+        col_size = np.prod(self.reordered_shape[self.axisptr :])
+        return (row_size, col_size)
+
+    @property
+    def reordered_shape(self):
+        return np.array(self.shape)[self.axis_order]
```

Comment:
Nit: We shouldn't use NumPy for manipulating shapes, it's a lot slower than using Python directly, and it sort-of implies heavy lifting. For example, I'd replace
```
np.array(self.shape)[self.axis_order]
```

with
```
tuple(self.shape[i] for i in axis_order)
```

Reply:
Thanks, this is helpful.